test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -6251,7 +6327,7 @@ TEST_F(AtenXlaTensorTest, TestMaxPool3DIncompleteAttributes) {
           });
 
           ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
-          ExpectCounterChanged("xla::max_pool3d_with_indices",
+          ExpectCounterChanged("xla::max_pool3d",
```

Comment:
Remind me to make a pass on this file and move those test before returning from function 😄 
No need to do it here.